packages/fluentui/docs/src/components/ExampleSnippet/ExampleSnippet.tsx

```diff
@@ -27,10 +29,14 @@ const ExampleSnippet: React.FunctionComponent<ExampleSnippetProps> = props => {
   const string = value || renderElementToJSX(element, !isFunctionWithoutValue);
 
   return (
-    <div style={rootStyle}>
-      <CodeSnippet value={string} fitted />
-      {element && <div style={renderedStyle}>{element}</div>}
-    </div>
+    <ProviderConsumer
+      render={({ siteVariables }) => (
+        <div style={rootStyle(siteVariables)}>
+          <CodeSnippet value={string} fitted />
+          {element && <div style={renderedStyle}>{element}</div>}
+        </div>
+      )}
+    />
```

Comment:
```suggestion
        <Box styles={rootStyles}>
          <CodeSnippet value={string} fitted />
          {element && <div style={renderedStyle}>{element}</div>}
        </Box>
```

You use `Box` component for this, then please also change:

```diff
-const rootStyle = siteVariables => ({
+const rootStyle = ({ siteVariables }) => ({
```

Reply:
`const rootStyle = ({ theme: { siteVariables } }) => ({`